index.html

```diff
@@ -0,0 +1,263 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Quiz Works</title>
+	<link rel="stylesheet" href="quiz.css">
+</head>
+<body>
+		
+		<article class="salescard">
+			<header class="scheader">
+				<h1>Spanky Cheese</h1>
+			</header>
+
+				<section class="section1">
```

Comment:
Dont add classes you aren't using for a css selector. Also make sure your classes are somewhat semantic.